packages/manager/apps/dedicated/client/app/app.routes.js

```diff
@@ -3,7 +3,7 @@ angular.module('App').config(
     $stateProvider.state('app', {
       abstract: true,
       resolve: {
-        currentUser: /* @ngInject */ (User) => User.getUser(),
+        currentUser: /* @ngInject */ (ducUser) => ducUser.getUser(),
```

Comment:
Maybe this will no longer be necessary with https://github.com/ovh/manager/pull/4080

Reply:
It's still required because of getUser in ducUser service does more than just fetching user. It creates a User object with user and certificate details.

Reply:
ducUser service can be modified to use user from the environment. This can be taken later when both of these are merged into develop branch.